release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1930,6 +1935,45 @@ With this release, the CCO no longer checks for the nonexistent permission.
 
 * Previously, proxy information that was set in the `install-config.yaml` file was not applied to the bootstrap process. With this release, proxy information is applied to bootstrap ignition data, which is then applied to the bootstrap machine. (link:https://issues.redhat.com/browse/OCPBUGS-12890[*OCPBUGS-12890*])
 
+* Previously, the {ibm-power-server-name} platform had no Dynamic Host Configuration Protocol (DHCP) network name, the DHCP resource was not deleted. With this release, a check looks for any DHCP resources with an `ERROR` state and deletes them so that this issue no longer occurs. (link:https://issues.redhat.com/browse/OCPBUGS-35224[*OCPBUGS-35224*])
+
+* Previously, when creating a Container-Native Storage API (CAPI) cluster on installer-provisioned infrastructure for {ibm-power-server-name}, the load balancer would become busy and stall. With this release, you can use the 'AddIPToLoadBalancerPool' command in a `PollUntilContextCancel' loop to restart the load balancer. (link: https://issues.redhat.com/browse/OCPBUGS-35088[*OCPBUGS-35088*])
+
+* Previously, an installer-provisioned installation on a bare-metal platform when Federal Information Processing Standards (FIPS) nodes are enabled caused installation failures. With this release, the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-34985[*OCPBUGS-34985*])
+
+* Previously, when creating an install configuration for an installer-provisioned installation on {ibm-power-server-name}, the survey stopped if the administrator did not enter a command on the {oc-first}. The survey stopped because no default region was set in the `install-config` survey. With this release, the issue is resolved.
+(link:https://issues.redhat.com/browse/OCPBUGS-34728[*OCPBUGS-34728*])
+
+* Previously, Solid state drives (SSD) using Serial ATA Solid State Drive (SATA) hardware were identified as removable. The Assisted Installer for {product-title} reported that no eligible disks were found and the installation stopped. With this release, removable disks are eligible for installation. (link:https://issues.redhat.com/browse/OCPBUGS-34652[*OCPBUGS-34652*])
```

Comment:
SATA is common enough to not spell out imo. If you do, it's short for "Serial AT Attachment"
```suggestion
* Previously, solid state drives (SSD) using SATA hardware were identified as removable. The Assisted Installer for {product-title} reported that no eligible disks were found and the installation stopped. With this release, removable disks are eligible for installation. (link:https://issues.redhat.com/browse/OCPBUGS-34652[*OCPBUGS-34652*])
```

Reply:
:heavy_check_mark: